examples/boltzmann_machine.py

```diff
@@ -15,7 +15,7 @@
 from dwave_networkx import zephyr_coordinates, zephyr_four_color, zephyr_graph
 from torch.optim import SGD
 
-from dwave.plugins.torch.boltzmann_machine import GraphRestrictedBoltzmannMachine as GRBM
+from dwave.plugins.torch.models.boltzmann_machine import GraphRestrictedBoltzmannMachine as GRBM
```

Comment:
Should we perhaps put `GraphRestrictedBoltzmannMachine` directly under the `models` namespace? This is currently being suggested in the DVAE PR for the `DiscreteVariationalAutoencoder` class.
```python
from dwave.plugins.torch.models import GraphRestrictedBoltzmannMachine as GRBM
```